doc/debugger.rst

```diff
@@ -922,6 +935,8 @@ Error codes
 +--------+------------------------------------------------------------------+
 | 0x03   | Not found (e.g. invalid breakpoint index)                        |
 +--------+------------------------------------------------------------------+
+| 0x04   | AppRequest error (i.e. request callback returned -1)             |
```

Comment:
Mostly I wanted to distinguish that the error was caused by user code (i.e. the target) rather than Duktape, none of the other types seemed to fit.  I guess Unspecified works though, you still get the string text either way.